docs/layouts/shortcodes/generated/prometheus_push_gateway_reporter_configuration.html

```diff
@@ -48,7 +48,7 @@
             <td><h5>randomJobNameSuffix</h5></td>
             <td style="word-wrap: break-word;">true</td>
             <td>Boolean</td>
-            <td>Specifies whether a random suffix should be appended to the job name.</td>
+            <td>Specifies whether a random suffix should be appended to the job name. Job at least have two container(JobManager and TaskManager), enabled it can diff each other.</td>
```

Comment:
This description is somewhat incorrect. This config adds different random suffix for different reporter instances, but it is not necessary that each task manager has its own metrics reporter (although for most of the time it is the case). For example a mini cluster only has one metrics reporter (see `MiniCluster#start`).

Reply:
It can even be that 1 task manager has multiple prometheus reporters ;)

Reply:
I see, thx. I'll close the merge request